src/index.js

```diff
@@ -1,10 +1,44 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable no-plusplus */
+/* eslint-disable no-console */
+
 import './style.css';
-import WebpackLogo from './img/webpack-logo.svg'
+import Task from './task';
+import ThreeVerticalDots from './img/3-vertical-dots.svg';
+
+class ToDo {
+  constructor() {
+    this.tasks = localStorage.tasks ? JSON.parse(localStorage.tasks) : [];
+  }
+
+  addTask(description) {
+    this.tasks.push(new Task(description, this.tasks.length));
+    this.updateLocalStorage();
+  }
+
+  updateLocalStorage() {
+    localStorage.tasks = JSON.stringify(this.tasks);
+  }
 
-const h1 = document.getElementsByTagName('h1')[0]
-h1.classList.add('new-style');
+  renderList() {
+    const list = document.getElementById('my-list');
+    list.innerHTML = '';
+    for (const task of this.tasks) {
+      list.innerHTML += `
+      <li>
+        ${task.description}
+      </li>
+      `;
+    }
+  }
+}
 
-const logo = document.createElement('img');
-logo.src = WebpackLogo;
+const myToDoList = new ToDo();
+if (localStorage.tasks === undefined) {
+  myToDoList.addTask('My first task');
+  myToDoList.addTask('My second task');
+  myToDoList.addTask('My third task');
```

Comment:
Let's create an array and iterate over the array as stated in the project instructions.
Screenshot 👇 
![image](https://user-images.githubusercontent.com/31897434/129821664-80a2c77d-06e8-42eb-b25e-074964c48790.png)